pbspro.spec

```diff
@@ -105,6 +105,7 @@ BuildRequires: libical-devel
 BuildRequires: ncurses-devel
 BuildRequires: perl
 BuildRequires: postgresql-devel
+BuildRequires: postgresql-contrib
```

Comment:
Please update pbspro.spec.in as well.

Reply:
Thanks @mike0042 . I have updated PBSPro.spec.in with the changes that I have done for psbpro.spec in the latest commit.